code/game/objects/items/weapons/storage/belt_vr.dm

```diff
@@ -1,4 +1,58 @@
 /obj/item/weapon/storage/belt
 	sprite_sheets = list(
 		SPECIES_TESHARI = 'icons/mob/species/seromi/belt.dmi',
-		SPECIES_WEREBEAST = 'icons/mob/species/werebeast/belt.dmi')
\ No newline at end of file
+		SPECIES_WEREBEAST = 'icons/mob/species/werebeast/belt.dmi')
+		
+/obj/item/weapon/storage/belt/explorer
+	name = "explorer's belt"
+	desc = "A versatile belt with several pouches. It can hold a very wide variety of items, but less items overall than a dedicated belt. Still, it's useful for any explorer who wants to be prepared for anything they might find."
+	icon_state = "explorer_belt"
+	icon = 'icons/obj/clothing/belts_vr.dmi'
+	icon_override = 'icons/mob/belt_vr.dmi'
+	storage_slots = 5	//makes it strictly inferior to any specialized belt as they have seven slots, but it's far more versatile
+	max_w_class = ITEMSIZE_NORMAL	//limits the max size of thing that can be put in, so no using it to hold five laser cannons
+	max_storage_space = ITEMSIZE_COST_NORMAL * 5
+	can_hold = list(
+		/obj/item/weapon/grenade,
+		/obj/item/weapon/tool,
+		/obj/item/weapon/weldingtool,
+		/obj/item/weapon/pickaxe,
+		/obj/item/device/multitool,
+		/obj/item/stack/cable_coil,
+		/obj/item/device/analyzer,
+		/obj/item/device/flashlight,
+		/obj/item/weapon/cell,
+		/obj/item/weapon/gun,
+		/obj/item/weapon/material,
+		/obj/item/weapon/melee,
+		/obj/item/weapon/shield,
+		/obj/item/ammo_casing,
+		/obj/item/ammo_magazine,
+		/obj/item/weapon/storage/box/beanbags,
+		/obj/item/weapon/storage/box/shotgunammo,
+		/obj/item/weapon/storage/box/shotgunshells,
```

Comment:
These being allowed means that explorers will be able to empty them and refill them with any number of items that fit in a box, bypassing both the belt item restrictions and the belt item slot limit.

Reply:
Shoot. Good point. I'll swap them for the shotgun speedloaders instead.